lib/src/data/net_stream.rs

```diff
@@ -16,6 +17,8 @@ pub enum NetStream {
     Http(HttpStream),
     #[cfg(feature = "tls")]
     Https(HttpsStream),
+    #[cfg(unix)]
+    UnixHttp(UnixSocketStream),
```

Comment:
I don't like this, but it is needed to make [concrete_stream](https://github.com/SergioBenitez/Rocket/pull/594/files#diff-58c10788d95c4f599ead66631ae49f04R110) work.

Reply:
Yes, it's all very frustrating.